controllers/profileRoutes.js

```diff
@@ -0,0 +1,43 @@
+const router = require('express').Router();
+const sequelize = require('../config/connection');
+const { Event, User, Task } = require('../models');
+const withAuth = require('../utils/auth');
+
+router.get('/', withAuth, (req, res) => {
+    Event.findAll({
+      where: {
+        // use the ID from the session
+        user_id: req.session.user_id
+      },
+      attributes: [
+        'id',
+        'name',
+        'event_date',
+        'event_address',
+        'event_owner'
+      ],
+      include: [
```

Comment:
Event does not have a reference to Task.  I think you would need to get Task separately for this user.

Reply:
Discuss further tomorrow.